WEB_SERVICE/WEB/_HIGH/LDAP_INJECTION/recommendation.md

```diff
@@ -0,0 +1,131 @@
+- __Parameterized Queries:__ Use parameterized queries instead of direct concatenation to construct LDAP queries.
```

Comment:
can you please add a sentence before this :  something like to prevent ....

Reply:
Added.